src/Calculator/Controls/EquationTextBox.cpp

```diff
@@ -421,6 +423,11 @@ bool EquationTextBox::RichEditHasContent()
 
 void EquationTextBox::OnRichEditMenuOpened(Object ^ /*sender*/, Object ^ /*args*/)
 {
+    if (m_richEditBox != nullptr)
+    {
+        VisualStateManager::GoToState(m_richEditBox, "Focused", false);
```

Comment:
I can still repro #1153 even with this change

Reply:
huh...i can't repro it. Is it the same repro steps?

Reply:
Yeah it was the repro steps for right clicking an unfocused textbox but only repros in dark mode,  I can double check

Reply:
I removed the this bug from the PR and moved it to it's own PR.